### ISSUE TITLE
Extract wrong parameters from methods

### ISSUE THREAD

**Issue by bsesso** [Kuniri/kuniri#170]
```ruby
class Bird    $
  def preen   $
    puts "I am cleaning my feathers."  
  end  
  def fly  
    puts "I am flying."  
  end  
end  
  
class Penguin < Bird  
  def fly  
    puts "Sorry. I'd rather swim."  
  end  
end  
  
p = Penguin.new  
p.preen  
p.fly  
```

**Comment by rodrigosiqueira** [comment]
Nice catch! Let's work on it next week :)

**Comment by bsesso** [comment]
Fixed on #212
